src/App.tsx

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
-import { SignIn, About } from "@components/index";
+import { About, SignIn, SignUp } from "@components/index";
 import { Landing, Pages } from "./pages";
-
 import { urlFormat } from "./utils/urlFormat";
```

Comment:
Could be '@/pages' and '@/utils' right? Like in `signIn`

Reply:
True, fixed!